yarn.lock

```diff
@@ -1617,7 +1617,7 @@ eslint-plugin-flowtype@^2.25.0:
   dependencies:
     lodash "^4.15.0"
 
-"eslint-plugin-react-internal@file:./eslint-rules":
+"eslint-plugin-react-internal@file:./scripts/eslint-rules":
```

Comment:
This seems like it should've been a part of my previous merged PR. Not sure why it didn't get included there.